.github/workflows/eco-gotests-integration.yml

```diff
@@ -0,0 +1,43 @@
+name: eco-gotests integration
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
```

Comment:
We want to run similar check for releases as well. 
For instance:
We sync eco-gotests release-4.15 with eco-goinfra release-4.15